build/windows/msi/build-updates-disabled.sh

```diff
@@ -11,7 +11,7 @@ SCRIPT_DIR=$( pwd )
 cd "../../../VSCode-win32-${VSCODE_ARCH}/resources/app"
 
 cp product.json product.json.bak
-cat product.json.bak | jq "del(.updateUrl)" > product.json
+jq "del(.updateUrl)" product.json.bak > product.json
```

Comment:
Can you update this block so it uses the same `jsonTmp` style? Thx

Reply:
Done.